src/wiki/static/wiki/js/diff.js

```diff
@@ -13,11 +17,11 @@ function get_diff_json(url, put_in_element) {
                                                   '<dd>'+data.other_changes[i][1]+'</dd>'  ));
           }
         }
-        put_in_element.find('.diff-container').show('fast', function() {put_in_element.collapse('show');});
-        $(put_in_element).parentsUntil('.card-group').find('.progress').detach();
+        $(put_in_element).find('.diff-container').show('fast', function() {$(put_in_element).collapse('show');});
+        $(put_in_element).parent().find('.progress').detach();
       });
-    } else {
-      put_in_element.find('.diff-container').show('fast', function() {put_in_element.collapse('toggle');});
-    }
-  });
-}
+    });
+  }else {
+    $(put_in_element).find('.diff-container').show('fast', function() {$(put_in_element).collapse('toggle');});
+  }
+});
```

Comment:
THANKS for fixing this, I noticed too that something had changed in the structure and got stuck. :100: